groupease-client/src/app/auth/auth.service.ts

```diff
@@ -30,7 +34,14 @@ export class AuthService {
     this.auth0.parseHash((err, authResult) => {
       if (authResult && authResult.accessToken && authResult.idToken) {
         this.setSession(authResult);
-        this.router.navigate([this.homeRoute]);
+        this.userService.saveCurrentUser()
+          .take(1)
+          .subscribe(
+            () => {
+              this.router.navigate([this.dashboardRoute]);
+            }
+          );
```

Comment:
On signing in, we call the new save user endpoint to ensure the user data is current and saved into the DB.

We then route to the dashboard view.